contracts/staking/interfaces/IValidatorManager.sol

```diff
@@ -101,6 +117,17 @@ interface IValidatorManager {
      */
     event ValidationPeriodEnded(bytes32 indexed validationID, ValidatorStatus indexed status);
 
+    /**
+     * @notice Verifies and sets the initial validator set for the chain through a P-Chain
+     * SubnetConversionMessage.
+     * @param subnetConversionData The subnet conversion message data used to recompute and verify against the subnetConversionID.
+     * @param messsageIndex The index that contains the SubnetConversionMessage Warp message in the transaction.
```

Comment:
```suggestion
     * @param messsageIndex The index that contains the SubnetConversionMessage Warp message containing the subnetConversionID to be verified against the provided {subnetConversionData}
```
Small suggestion to link the `subnetConversionData` param with the Warp message retreived from `messageIndex`.